frontend/lib/laletterbuilder/route-info.ts

```diff
@@ -61,4 +61,5 @@ export const getLaLetterBuilderJumpToTopOfPageRoutes = () => [
 
 export const getLaLetterBuilderRoutesForPrimaryPages = () => [
   LaLetterBuilderRouteInfo.locale.home,
+  LaLetterBuilderRouteInfo.locale.chooseLetter,
```

Comment:
I think the only thing this is used for is to decide whether the browser should always jump to the top of the page (I seem to recall Atul saying this had been a problem on some browsers before). This is different from the CSS class `jf-laletterbuilder-section-primary`, which refers to the ordering of sections. All of which is to say, I don't think it's necessary here unless there's another function it's serving?